posthog/hogql/query.py

```diff
@@ -200,6 +202,8 @@ def execute_hogql_query(
                     team_id=team.pk,
                     readonly=True,
                 )
+            except (ExposedHogQLError, ExposedCHQueryError) as e:
+                raise ValidationError(str(e), getattr(e, "code_name", None))
```

Comment:
There might be reason we are not re-raising this here as `ValidationError` (if you do raise, it's also a good idea to use `raise ... from e` by the way)

You can see in https://github.com/PostHog/posthog/blob/ed4719727917c8c64257f7c1dbe803691ed1b554/posthog/api/query.py#L91-L92 how we do it for the query endpoint. So if there problem is for replay, maybe do it somewhere closer to the API level there?

Reply:
If you're looking for where this gets called in Replay land it's most likely coming from this line:
https://github.com/PostHog/posthog/blob/00dd4e8028b12e981e7d70d6a2ad05766b6a69db/frontend/src/scenes/session-recordings/playlist/sessionRecordingsPlaylistLogic.ts#L365

Reply:
Sure, thanks for your help @daibhin!
I'll definitely check it out